scheduler/src/cook/kubernetes/api.clj

```diff
@@ -189,7 +189,7 @@
   [^V1Node node]
   ; In the case of nil, we have taints-on-node == [], and we'll map to no-pool.
   (let [taints-on-node (or (some-> node .getSpec .getTaints) [])
-        cook-pool-taint (filter #(= "cook.pool" (.getKey %)) taints-on-node)]
+        cook-pool-taint (filter #(= "cook-pool" (.getKey %)) taints-on-node)]
```

Comment:
Should we make this be a constant?

Reply:
Why? Its another level of indirection a constant we will never change because it is API. The constant is also used in python and clojure.